src/components/UserInfo.js

```diff
@@ -1,19 +1,26 @@
-export default class UserInfor {
-  constructor(nameSelector, descriptionSelector) {
+export default class UserInfo {
+  constructor({ nameSelector, descriptionSelector, avatarSelector }) {
     this._nameElement = document.querySelector(nameSelector);
     this._descriptionElement = document.querySelector(descriptionSelector);
+    this._avatarElement = document.querySelector(avatarSelector);
   }
 
   getUserInfo() {
-    const userInfo = {
+    return {
       name: this._nameElement.textContent,
       description: this._descriptionElement.textContent,
     };
-    return userInfo;
   }
 
-  setUserInfo(name, description) {
+  setUserInfo({ name, description, avatar }) {
     this._nameElement.textContent = name;
     this._descriptionElement.textContent = description;
+    this._avatarElement.src = avatar;
```

Comment:
`  this._avatarElement.src = avatar;` is duplicated here and in `setAvatar`


Please, use `setAvatar` here or delete `setAvatar`